backend/lcfs/web/api/fuel_supply/validation.py

```diff
@@ -18,6 +18,19 @@ def __init__(
     async def check_duplicate(self, fuel_supply: FuelSupplyCreateUpdateSchema):
         return await self.fs_repo.check_duplicate(fuel_supply)
 
+    async def check_duplicate_of_prev_data(
```

Comment:
please add commenting to this  method describing its purpose, will be hard to track without context.